applications/app/components/MyComponent.scala

```diff
@@ -0,0 +1,30 @@
+package components
+
+import components.core.{ComponentCss, Css, HtmlComponent}
+import model.ApplicationContext
+import play.twirl.api.Html
+
+case class MyComponent(implicit context: ApplicationContext) extends HtmlComponent {
+
+  val myTitle: MyTitle = MyTitle("This is the title")
+  val myText: MyText = MyText("This is the text")
+
+  override def html: Html = views.html.prototype.MyComponent()(myTitle.html, myText.html)
+  override def componentCss: Seq[Css] = Seq(
+    ComponentCss.load("component-myComponent")
+  )
+}
```

Comment:
This is how one simply defines a component:
- all subcomponents needs to be explicitly specified
- `html` from a twirl template
- `css` from a set of css files

It is quite flexible: parameters can be used to combine and pass a component to another, etc...